templates/main/index.html.ep

```diff
@@ -44,6 +44,7 @@
             </div>
             <div class="form-group">
                 <strong>Misc</strong>
+                <label><input value="1" name="show_tags" type="checkbox" id="filter-show-tags"> Show tags</label>
```

Comment:
Can you add a *title*, e.g. "Show tags on builds that have been tagged in the corresponding group overview comments" ? Or immediately do

```
… data-toggle="tooltip" \
    title="Show tags on builds that have been tagged in the corresponding \
    group overview comments<br>See \
    <%= link_to here => "https://progress.opensuse.org/projects/openqav3/wiki/Wiki#Build-tagging-and-keeping-important-builds-gh591" %> \
    for details…
```